stream-chat-android-client/src/main/java/io/getstream/chat/android/client/controllers/ChannelControllerImpl.kt

```diff
@@ -72,6 +72,10 @@ import io.getstream.chat.android.client.utils.observable.ChatObservable
 import io.getstream.chat.android.client.utils.observable.Disposable
 import java.io.File
 
+@Deprecated(
+    message = "The ChannelControllerImpl has been replaced by ChannelClient",
+    replaceWith = ReplaceWith("ChannelClient")
+)
 internal class ChannelControllerImpl(
```

Comment:
We can rename this class to `ChannelClient`, and have it implement the now deprecated `ChannelController` interface. Then we can remove that interface later on.